osmnames/import_osm/01_delete_unusable_entries.sql

```diff
@@ -1,9 +1,10 @@
---cleanup unusable entries
-DELETE FROM osm_polygon_tmp WHERE (name <> '' OR name_fr <> '' OR name_en <> '' OR name_de <> '' OR name_es <> '' OR name_ru <> '' OR name_zh <> '') IS FALSE;
-DELETE FROM osm_point_tmp WHERE (name <> '' OR name_fr <> '' OR name_en <> '' OR name_de <> '' OR name_es <> '' OR name_ru <> '' OR name_zh <> '') IS FALSE;
-DELETE FROM osm_linestring_tmp WHERE (name <> '' OR name_fr <> '' OR name_en <> '' OR name_de <> '' OR name_es <> '' OR name_ru <> '' OR name_zh <> '') IS FALSE;
+-- cleanup unusable entries
+-- remove all rows where all names are empty or null values
+DELETE FROM osm_polygon_tmp WHERE (name = '' AND name_fr = '' AND name_en = '' AND name_de = '' AND name_es = '' AND name_ru = '' AND name_zh = '') IS NOT FALSE;
```

Comment:
This is, logically speaking, the same behaviour as before, but does it remove null values? (same comment for the two lines that follow)